src/data-generator/constants.ts

```diff
@@ -1,3 +1,4 @@
+export const COUNTRY = 'FAR';
```

Comment:
Do not set COUNTRY here as a constant, please as it means there is another location where country code must be hardcoded and managed in documentation.  

Can you use a fetch, to load client-config.js in the data generator script, then extract the country string from the window.config variable?